doc/api.yaml

```diff
@@ -492,15 +479,92 @@ paths:
           $ref: '#/components/responses/404'
         '500':
           $ref: '#/components/responses/500'
-
-
+  /ideas/{idea_id}/picture:
+    post:
+      tags:
+        - Ideas
```

Comment:
Check we have two duplicated Idea tags

Reply:
Tags changed to "idea" on all ressources involving idea

Reply:
you can also resolve discussion if it's resolved